### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
replaced a long time ago with the correct date (august 2017)

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4130

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -76,10 +76,11 @@ For more information about the Azure AD P1 and P2, see [Azure Active Directory p
 
 ### Turn on Modern authentication for your organization
 
-For most subscriptions modern authentication is automatically turned on, but if you purchased your subscription a long time ago, it might not be. This has to be turned on before MFA works appropriately with Office apps.
+For most subscriptions modern authentication is automatically turned on, but if you purchased your subscription before August 2017, it is likely that you will need to turn on Modern Authentication in order to get features like Multi-Factor Authentication to work in Windows clients like Outlook
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
For most subscriptions modern authentication is automatically turned on, but if you purchased your subscription before August 2017, it is likely that you will need to turn on Modern Authentication in order to get features like Multi-Factor Authentication to work in Windows clients like Outlook.
```

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 2]
```diff
@@ -76,10 +76,11 @@ For more information about the Azure AD P1 and P2, see [Azure Active Directory p
...
+
 
 1. In the Microsoft 365 admin center, in the left nav choose **Settings** \> **Org settings**.
-1. Under **Services** tab, choose **Modern authentication**, and in the **Modern authentication** pane, make sure **Enable Modern authentication** is selected. Choose **Save changes**.
+2. Under **Services** tab, choose **Modern authentication**, and in the **Modern authentication** pane, make sure **Enable Modern authentication** is selected. Choose **Save changes**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
2. Under the **Services** tab, choose **Modern authentication**, and in the **Modern authentication** pane, make sure **Enable Modern authentication** is selected. Choose **Save changes**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**